storage/src/main/java/org/apache/kafka/server/log/remote/metadata/storage/TopicBasedRemoteLogMetadataManager.java

```diff
@@ -515,6 +517,11 @@ public void startConsumerThread() {
         }
     }
```

Comment:
btw, `startConsumerThread` it's unused, can be removed?

Reply:
done